cargo-apk/src/apk.rs

```diff
@@ -225,13 +225,41 @@ impl<'a> ApkBuilder<'a> {
 
         let signing_key = self.manifest.signing.get(profile_name);
 
-        let signing_key = match (signing_key, is_debug_profile) {
-            (Some(signing), _) => Key {
+        let signing_key = if let Some(signing) = signing_key {
```

Comment:
What's your thought on letting the env vars take precedence over what was configured in `self.manifest.signing`?